serveradmin/servershell/views.py

```diff
@@ -1,90 +1,129 @@
 """Serveradmin - Servershell
 
-Copyright (c) 2019 InnoGames GmbH
+Copyright (c) 2020 InnoGames GmbH
```

Comment:
To save some work once a year ;D
`date.today().year`

Reply:
Because it harms readability to have doc strings concatenated with " + date.today().year + " and makes it possibly broken for automatic doc generators.

Reply:
I will close it for now. Feel free to get in touch with me. Thanks for the review btw!